### PR TITLE
Deleting some spaces

### PR DESCRIPTION
**Pull request by Stilky**
I inspected much of the CEMU source code and removed spaces that seemed unnecessary to me.

### CODE REVIEW

File: src/Cafe/HW/Latte/Renderer/Vulkan/VKRPipelineInfo.cpp [hunk 1]
```diff
@@ -3,6 +3,7 @@
 #include "Cafe/HW/Latte/Renderer/Vulkan/LatteTextureVk.h"
 #include "Cafe/HW/Latte/Renderer/Vulkan/RendererShaderVk.h"
 
+#include <glslang/Include/Types.h>
```
Comment (Stilky) [inline src/Cafe/HW/Latte/Renderer/Vulkan/VKRPipelineInfo.cpp:6]: I don't understand why it changed that for me when I didn't touch it.

File: src/Cafe/HW/Latte/Renderer/Vulkan/VulkanRenderer.cpp [hunk 2]
```diff
@@ -26,7 +26,7 @@
 
 #include "Cafe/HW/Latte/Core/LatteTiming.h" // vsync control
 
-#include <glslang/Public/ShaderLang.h>
+#include <glslang/Include/Types.h>
```
Comment (Stilky) [inline src/Cafe/HW/Latte/Renderer/Vulkan/VulkanRenderer.cpp:29]: Same for that

File: src/input/api/Wiimote/hidapi/HidapiWiimote.cpp [hunk 3]
```diff
@@ -32,8 +30,6 @@ std::vector<WiimoteDevicePtr> HidapiWiimote::get_devices() {
     for (auto it = device_enumeration; it != nullptr; it = it->next){
         if (it->product_id != WIIMOTE_PRODUCT_ID && it->product_id != WIIMOTE_MP_PRODUCT_ID)
             continue;
-        if (std::wcscmp(it->product_string, PRO_CONTROLLER_NAME) == 0)
```
Comment (Stilky) [inline src/input/api/Wiimote/hidapi/HidapiWiimote.cpp:35]: I never deleted this

File: src/input/api/Wiimote/hidapi/HidapiWiimote.cpp
[the remark this answers is not in the document]
Comment (Stilky) [reply src/input/api/Wiimote/hidapi/HidapiWiimote.cpp:1]: I don't know why it was changed, I didn't touch it

**Comment by Exzap** [comment]
Hey, I appreciate that you are trying to contribute. But when it comes to pure clean-up commits their value has to be weighted against the conflicts they can cause. In this case, merging would cause a conflict for #641 since some of the files that you cleaned up have been moved&reworked by it. Conflicts are only a minor annoyance, but empty lines are even less significant so with this in mind it's just not worth the hassle and I will unfortunately have to close this PR.

Additionally, as @capitalistspz already pointed out, you accidentally undid changes from an earlier commit. Mistakes can happen, but when you plan on submitting a PR you should review it first. Github shows you a preview of all the changes while creating the PR, make use of this to spot any potential mistakes or unwanted changes and fix those before submitting the PR.

**Pull request closed without merging**